frontend/src/features/common/commonSlice.js

```diff
@@ -148,6 +168,21 @@ export const commonSlice = createSlice({
         state.allTokensInfoState.error = action.payload;
         state.allTokensInfoState.info = {};
       });
+
+    builder
+      .addCase(getICNSName.pending, (state) => {})
+      .addCase(getICNSName.fulfilled, (state, action) => {
+        const address = action.payload?.address || "";
+        if (address?.length) {
+          let result = {
```

Comment:
if you are not using the variable multiple times, what is the use of declaring the variable? directly assign the object.